src/flag_gems/ops/cross_entropy_loss.py

```diff
@@ -302,7 +304,6 @@ def celoss_probability_bwd(
     inp_grad_ptr,
     label_smoothing,
     mean_num,
-    N,
     C,
```

Comment:
There is possibility that `N` is critical to the choice of `BLOCK_C` and `BLOCK_D` and thus should be used in autotunner key.